### PR TITLE
Fix signature verification

### LINKED ISSUE #4: Respone verification always returns false
Branch `server_response_verification`
I'm guessing the template is wrong or something else is bugging out.

### PR DESCRIPTION
**Pull request by robbertkl**
I noticed a mistake in the signature verification code and fixed it according to forge's own examples.

I guess this is why response verification is disabled?

https://github.com/BunqCommunity/BunqJSClient/blob/66be2aa45ca17352a14293f38ef4444c31a02c7f/src/ApiAdapter.ts#L369-L377

Also, together with re-enabling response verification, this will probably fix #4, unless there are other things preventing it from successfully verifying the response.

### CODE REVIEW

**Comment by Crecket** [comment]
Thanks! I'll look into this later today

**Comment by Crecket** [comment]
It seems to be working, I'll add some tests when I get home and then create a new release

**Comment by robbertkl** [comment]
Great, thanks

File: .all-contributersrc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      "contributions": [
+        "code",
+        "design",
+        "doc",
+        "infra",
+        "review",
+        "platform",
+        "question",
+        "bug"
+      ]
```
Comment (Crecket) [inline .all-contributersrc]: Need to add all missing contributors before merge

**Comment by robbertkl** [comment]
@Crecket Haven't checked if you're testing this, but make sure to verify the signature verification is also working with binary data, e.g. `GET /v1/attachment-public/<uuid>/content`. Since the forge verification happens on a string, there's a chance the binary data got encoded as a UTF-8 string somewhere along the way.

**Comment by Crecket** [comment]
Yeah I'm working on this now 👍 I actually just created an issue for it so I'll disable it for now and fix it later this week when I have more time

**Pull request merged**